src/Command/ValidateAppCommand.php

```diff
@@ -35,9 +37,19 @@ protected function execute (InputInterface $input, OutputInterface $output) : in
 		$io = new TorrStyle($input, $output);
 		$io->title("Hosting: Validate App");
 
-		$event = new ValidateAppEvent($io);
-		$this->dispatcher->dispatch($event);
-		$failedCheck = $event->getFailedCheck();
+		try
+		{
+			$event = new ValidateAppEvent($io);
+			$this->dispatcher->dispatch($event);
+			$failedCheck = $event->getFailedCheck();
+		}
+		catch (\Exception $exception)
+		{
+			$this->logger->error("Failed to validate app", [
+				"exception" => $exception,
+			]);
+			$failedCheck = $exception->getMessage();
```

Comment:
Could this potentially leak sensitive information? If so: Do we care (within the context of the CI) about it? And/or could we feasible do something about it?

Reply:
As this is supposed to be run from the CI, showing useful information is part of the intended usage here.
And an exception should never carry any sensitive information, especially not in the default message.